Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserComponent.h

```diff
@@ -127,12 +129,17 @@ namespace AzToolsFramework
 
             AzFramework::SocketConnection::TMessageCallbackHandle m_cbHandle = 0;
 
+            AzQtComponents::StyledBusyLabel* m_styledBusyLabel;
+
             //! Notify to start the query thread
             void NotifyUpdateThread();
 
             void HandleFileInfoNotification(const void* buffer, unsigned int bufferSize);
 
-            AzQtComponents::StyledBusyLabel* m_styledBusyLabel;
+            //////////////////////////////////////////////////////////////////////////
+            // AssetBrowserFileCreationNotificationBus
+            void HandleAssetCreatedInEditor(const AZStd::string_view assetPath, const AZ::Crc32& creatorBusId /*= AZ::Crc32()*/) override;
+            //////////////////////////////////////////////////////////////////////////
         };
     }
 } // namespace AssetBrowser
```

Comment:
Nit: incorrect namespace comment (above should be AssetBrowser, thsis should be AzToolsFramework)

Reply:
Done!